src/pybind/mgr/volumes/fs/operations/volume.py

```diff
@@ -260,6 +261,32 @@ def get_pending_subvol_deletions_count(fs, path):
     return {'pending_subvolume_deletions': num_pending_subvol_del}
 
 
+def get_all_pending_clones_count(self, mgr, vol_spec):
+    pending_clones_cnt = 0
+    index_path = ""
+    
+    fs_map = mgr.get('fs_map')
+    for fs in fs_map['filesystems']:
+        volname = fs['mdsmap']['fs_name']
+        try:
+            with open_volume(self, volname) as fs_handle:
+                try:
+                    with open_clone_index(fs_handle, vol_spec) as index:
+                        index_path = index.path.decode('utf-8')
+                        pending_clones_cnt = pending_clones_cnt \
+                                             + len(listdir(fs_handle, index_path, 
+                                                           filter_entries=None, filter_files=False))
+                except IndexException as e:
+                    if e.errno == -errno.ENOENT:
+                        continue
+                    raise e
```

Comment:
Same as below comment. Let's have single try except block for both `open_volume` and `open_clone_index` and convert `IndexException` to `VolumeException` and caller can handle only `VolumeException`

Reply:
Done!